recipes/imagepy/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "imagepy" %}
+{% set version = "0.16" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "511556789365fffa4827f771ca246d10bbb7fd7a07240d6a29c39b3e44e83e1e" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.tar.gz'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ hash_value }}
+
+build:
+  number: 0
+  script: python setup.py install  --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - scikit-image >=0.12
+    - shapely >=1.0
+    - wxpython >=4.0
+    - numba >=0.29
+
+test:
+  imports:
+    - imagepy
+
+about:
+  home: https://github.com/Image-Py/imagepy
+  license: BSD 3-clause
+  license_family: BSD
+  license_file: ''
+  summary: interactive python image-processing plugin framework
+  description: Image process framework based on plugin like imagej, it is esay to glue with scipy.ndimage, scikit-image, opencv, simpleitk, mayavi...and any libraries based on numpy
```

Comment:
Would suggest wrapping the `description`. Here is a [good example]( https://github.com/conda-forge/contact_map-feedstock/blob/ade8af18ac21c30ffcafdaa6f0c9a22ea86a468c/recipe/meta.yaml#L40-L43 ).